.github/workflows/nebula-ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        # test against JDK 8
-        java: [ 8 ]
+        # test against JDK 11 
+        java: [ 11 ]
```

Comment:
I would add other popular and/or upcoming flavors: 17 and 21.

Reply:
Done